Source/WTF/wtf/FastMalloc.h

```diff
@@ -682,8 +682,9 @@ using __thisIsHereToForceASemicolonAfterThisMacro UNUSED_TYPE_ALIAS = int
 #define WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR_IMPL(T) \
 void operator delete(T* object, std::destroying_delete_t, size_t size) { \
     ASSERT(sizeof(T) == size); \
+    auto checkedPtrCountWithoutThreadCheck = object->checkedPtrCountWithoutThreadCheck(); \
```

Comment:
Loading `checkedPtrCountWithoutThreadCheck` before we run the destructor changes the semantics of CheckedPtr. Specifically, this case (using CheckedPtr as a weak back pointer to your owner) is supported by the current implementation, but will crash after this patch:

```
class A {
    B m_b;
}

class B {
    CheckedPtr<A> m_a;
}

```

If the change passes tests, maybe it's OK?

On the other hand, do we need to make this change, which limits one use case for CheckedPtr?

Reply:
An alternative approach to consider is using a #pragma to ignore the warning in this case. We know the memory is "uninitialized" from the perspective of constructor initialization, and that's on purpose.

Reply:
There is a PR doing this: https://github.com/WebKit/WebKit/pull/29087. Can we land it?

Reply:
@obyknovenius Now that https://github.com/WebKit/WebKit/pull/29087 has been landed, I think we can close this one, right?